src/ColorContextMenu-Core.package/ColorChooserMorph.class/instance/buildDiscreteMenu.attribute.levels..st

```diff
@@ -0,0 +1,13 @@
+menus
+buildDiscreteMenu: menuFactory attribute: aSymbol levels: levels
+
+	| menu setter |
+	self flag: 'Warum heißt es hier menuFactory und nicht aRadialMenuMorph?'.
```

Comment:
`#buildDiscremeteMenu:...` is also used for the construction of the context menu. The goal is to make them completely polymorphic!

```suggestion
```